modules/timBidAdapter.js

```diff
@@ -42,12 +42,12 @@ function formatAdMarkup(bid) {
 
 export const spec = {
   code: BIDDER_CODE,
-  aliases: ['tim'], // short code
 
   isBidRequestValid: function(bid) {
     if(bid.params && bid.params.publisherid && bid.params.placementCode){
       return true;
     }
+    utils.logError("error bid: " + JSON.stringify(bid));
```

Comment:
this doesn't add enough info.
please make a clear check and log as to what param is missing